apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -124,6 +124,7 @@ export class PathSelectorTable {
     const row = $(event.target).closest('tr').get(0) || event.target;
     const url = row.dataset['apiUrl'];
     const pathType = row.dataset['pathType'];
+    this.activateFavorite(row);
```

Comment:
I'm not 100% sold on this, so I could be convinced to get rid of it, but visually it's nice to have things activate if you click on them.